README.md

```diff
@@ -78,11 +78,17 @@ Additionally an OpenCL Installable Client Driver (ICD) for your compute device i
 
 Run the GUI application:
 ```sh
-./ppm-gui
+./GUI/ppm-gui
+```
+
+In order to make the GUI application appear in the system application menu and 'Open with' context menus, link the `ppm-gui` application to a location that is on PATH, e.g. `~/.local/bin`, and install the provided .desktop file. This can be achived by running the following in the repository root:
+```bash
+ln -s `realpath ./GUI/ppm-gui` $HOME/.local/bin
```

Comment:
I guess, that will change, once we will have the full pip way, but OK for now.

Reply:
Yes, this is meant to be temporary.